### DISCUSSION TITLE
Spring Boot 3.1.X Version Compatibility

### DISCUSSION THREAD

**Discussion by tbarrett-RH** [awspring/spring-cloud-aws#863] [Q&A] [answered]
It appears from the dependency matrix in the README that spring-cloud-aws 3.0.1 is not compatible with Spring Boot 3.1.x releases, but I wanted to confirm this. I am trying to upgrade to spring-boot 3.1.2, using spring cloud 2022.0.3. Applications without spring-cloud-aws are able to run fine, however applications with spring cloud aws fail to start:

Spring Boot [3.1.2] is not compatible with this Spring Cloud release train

There are multiple things that could cause this of course, but given the README does not list Spring Boot versions 3.1.x as compatible with spring cloud aws 3.0x I figured I would check here first.

**Comment by alikian** [comment, marked as answer]
I have an app with spring boot 3.1.2 and spring cloud aws 3.0.1 and it work without issue
https://github.com/alikian/spring-heroes/blob/main/backend/pom.xml
